fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateMaterializedViewStmt.java

```diff
@@ -291,84 +262,76 @@ public <R, C> R accept(AstVisitor<R, C> visitor, C context) {
         return visitor.visitCreateMaterializedViewStmt(this, context);
     }
 
-    public static void analyze(StatementBase stmt, ConnectContext session) {
-        new OldMVAnalyzerVisitor().visit(stmt, session);
-    }
+    public void analyze(ConnectContext context) {
+        QueryStatement queryStatement = this.getQueryStatement();
+        long originSelectLimit = context.getSessionVariable().getSqlSelectLimit();
+        // ignore limit in creating mv
+        context.getSessionVariable().setSqlSelectLimit(SessionVariable.DEFAULT_SELECT_LIMIT);
+        context.getSessionVariable().setSqlSelectLimit(originSelectLimit);
```

Comment:
why here? bug?

Reply:
no need `OldMVAnalyzerVisitor ` visitor any more because only `visitCreateMaterializedViewStmt ` is hit, just mov old visitCreateMaterializedViewStmt' codes here.